### DISCUSSION TITLE
How to get GitHub content

### DISCUSSION THREAD

**Discussion by adnan-qazii** [community/community#102694] [GitHub Education]
### Hi everyone,

any suggestions?

**Comment by darrehanrasool** [comment]
enroll with student id if you are college student enroll with student id to get student developers pack

**Comment by ettaboyle** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
